java/1-Two-Sum.java

```diff
@@ -3,16 +3,16 @@ public int[] twoSum(int[] nums, int target) {
         HashMap<Integer, Integer> prevMap = new HashMap<>();
         
         for (int i = 0; i < nums.length; i++) {
-            int num = nums[i];
-            int diff = target - num;
+            int diffNum = nums[i];
+            int diff = target - nums[i];
             
-            if (prevMap.containsKey(nums[i])) {
-                return new int[]{prevMap.get(num), i};
+            if (prevMap.containsKey(diffNum)) {
+                return new int[]{prevMap.get(diffNum), i};
             }
             
-            prevMap.put(target - num, i);
+            prevMap.put(diff, i);
```

Comment:
this line change makes sense!

Reply:
@mitchellirvin I see, I was kinda hesitant of the new variable name as well, thanks for clarifying!